frontend/public/components/factory/list-page.jsx

```diff
@@ -57,6 +59,14 @@ TextFilter.displayName = 'TextFilter';
 
 /** @augments {React.PureComponent<{ListComponent: React.ComponentType<any>, kinds: string[], flatten?: function, data?: any[], rowFilters?: any[]}>} */
 export class ListPageWrapper_ extends React.PureComponent {
+  componentDidMount() {
+    if (_.size(this.props.kinds) === 1) {
+      store.dispatch(UIActions.setTableObject(kindObj(this.props.kinds[0])));
```

Comment:
We should only request the table output for pages that need it (default list pages with no custom table columns).

We also shouldn't be watching resources in the default table since we're using the table data. It looks like we're opening a watch and not using the data.

Reply:
We probably want to add polling to this eventually.